modules/admin_manual/pages/maintenance/upgrade.adoc

```diff
@@ -35,20 +39,18 @@ the new ownCloud release. Ensure that they are all disabled before
 beginning the upgrade. After the upgrade is complete re-enable any which
 are compatible with the new release.
 
-CAUTION: Install unsupported apps at your own risk.
+CAUTION: Unsupported apps may disrupt your upgrade.
 
 [[upgrade-options]]
 == Upgrade Options
 
 There are three ways to upgrade your ownCloud server:
 
-1.  *(Recommended)* Perform a xref:maintenance/manual_upgrade.adoc[manual upgrade], using 
+1.  *(Recommended)* Perform a xref:maintenance/manual_upgrade.adoc[manual upgrade], using
 http://owncloud.org/install/[the latest ownCloud release].
-2.  Use xref:maintenance/package_upgrade.adoc[your distribution’s package manager], 
-in conjunction with our official ownCloud repositories. *Note:* This approach should not be used unattended 
+2.  *(Discouraged)* Use xref:maintenance/package_upgrade.adoc[your distribution’s package manager],
+in conjunction with our official ownCloud repositories. *Note:* This approach should not be used unattended
 nor in clustered setups.
-3.  Use xref:maintenance/update.adoc[the Updater App]. This is needed in scenarios where the admin does 
-not have access to the command line. It is recommended for shared hosting environments and for users who 
-want an easy way to track different release channels.
+We discourage upgrades with Linux Package Manager because you might encounter unwanted side effects.
```

Comment:
How about linking to further information on the unwanted side-effects?

Reply:
I have no knowledge about such side effects or any links to it.
@JKawohl @voroyam any info you can provide ?